inspirehep/dojson/conferences/fields/bd1xx.py

```diff
@@ -33,40 +35,70 @@
 @utils.for_each_value
 def acronym(self, key, value):
     """Conference acronym."""
+    def append_to_self(title):
+        """Make a dictionary and append to self."""
+        titles = {}
+        titles['title'] = title
+        titles['subtitle'] = value.get('b')
+        titles['source'] = value.get('9')
+        if titles:
+            self['titles'].append(titles)
+
     self['date'] = value.get('d')
     self['opening_date'] = value.get('x')
     self['closing_date'] = value.get('y')
     self['cnum'] = value.get('g')
-    self['place'] = value.get('c')
-    self['subtitle'] = value.get('b')
-    self['title'] = value.get('a')
+    self.setdefault('titles', [])
+    title = value.get('a')
+    if isinstance(title, (tuple, list)):
+        for t in title:
+            append_to_self(t)
+    else:
+        append_to_self(title)
+
+    if value.get('c'):
+        self.setdefault('address', [])
+        raw_addresses = utils.force_list(value.get('c'))
+        for raw_address in raw_addresses:
+            address = inspire_dojson_utils.parse_conference_address(raw_address)
+            self['address'].append(address)
+
     return utils.force_list(value.get('e'))
 
 
 @conferences.over('alternative_titles', '^711')
 @utils.for_each_value
 def alternative_titles(self, key, value):
-    """Alternative title."""
-    return value.get('a')
+    """Alternative title.
 
-
-@conferences.over('contact_person', '^270')
-@utils.for_each_value
-def contact_person(self, key, value):
-    """Contact person."""
-    if value.get('m'):
-        self['contact_email'] = value.get('m')
-    return value.get('p')
+    711__b is for indexing and will not be displayed.
+    """
+    titles = {}
+    titles['title'] = value.get('a')
+    titles['searchable_title'] = value.get('b')
+    return titles
```

Comment:
Consistency: in the rest of the functions this is usually written as
```python
return {
    'title': value.get('a'),
    'searchable_title': value.get('b')
}
```

Is there a reason to change style here?